### PR TITLE
doc: update encryption doc for rbd

### PR DESCRIPTION
**Pull request by humblec**
Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

### CODE REVIEW

**Comment by humblec** [comment]
@vasyl-purchel @Madhu-1 ptal.

**Comment by Madhu-1** [comment]
@humblec please update the PR template

**Comment by humblec** [comment]
@Madhu-1 ptal.

File: docs/deploy-rbd.md [hunk 1]
```diff
@@ -222,7 +215,13 @@ and `csi.storage.k8s.io/node-stage-secret-namespace`).
 To encrypt rbd volumes with LUKS you need to set encryption passphrase in
 secrets under `encryptionPassphrase` key and switch `encrypted` option in
 StorageClass to `"true"`. This is not supported for storage classes that already
```
Comment (Madhu-1) [inline docs/deploy-rbd.md:217]: I think this doc needs to be updated for KMS and secret based encryption. these secrets are not needed for if we are using vault based encryption

Comment (humblec) [reply docs/deploy-rbd.md:217]: @Madhu-1 Thats why I put it in `Encryption Configuration` section and not in `Encryption KMS configuration` section.

**Review by Madhu-1 — APPROVED** [review]

File: docs/deploy-rbd.md (hunk 1 above)
Comment (Madhu-1) [reply docs/deploy-rbd.md:217]: correct. sorry i didnt noticed it

Comment (Madhu-1) [reply docs/deploy-rbd.md:217]: we have two sections Encryption configuration and Encryption KMS configuration

the Encryption configuration should be the generic one and it should be the common configuration for both kms and secret-based encryption.  Like `Encryption KMS configuration` we can have one more section for `Encryption secret configuration`

In this PR we are adding secret based encryption details to `Encryption configuration` do we need this one?

Comment (Madhu-1) [reply docs/deploy-rbd.md:217]: @humblec I was waiting to get an answer to the above question.  not sure why you added ready-to-merge label.

Comment (humblec) [reply docs/deploy-rbd.md:217]: @Madhu-1 ah.. missed this or didnt notice it as this marked as resolved. Any way, `Encryption Configuration` is meant for `secret` in our current deployment doc. We could make it more explicit with an explicit heading or title though, if needed?

**Pull request merged**